demo/k8s-validating-admission-policy/README.md

```diff
@@ -5,7 +5,6 @@
 
 - Requires minimum Gatekeeper v3.14.0
 - Set `--experimental-enable-k8s-native-validation` in Gatekeeper deployments.
-- Set `--validate-template-rego=false` in Gatekeeper deployments if using Gatekeeper version 3.14.0 and later. This flag will be removed in v3.16.0, and will not be applicable in the future.
```

Comment:
since this is not versioned, can we keep this for now?

Reply:
Done.